backoffice/views/courses_views.py

```diff
@@ -144,6 +144,22 @@ def course_detail(request, course_key_string):
 
 
 @group_required('fun_backoffice')
+def users(request, course_key_string, action=None):
+    course = get_course(course_key_string)
+    course_info = get_course_infos([course])[0]
+
+    users = User.objects.filter(courseenrollment__course_id=course.id).order_by('username')
```

Comment:
Apparemment d'après Laurent pas besoin de fournir la liste. Générer le csv
suffit.

Parce que l'intérêt de ce ticket est de faire des recherches sur plein
d'utilisateurs à la fois.

Le lun. 30 mai 2016 08:53, Richard Moch <notifications@github.com> a écrit :

> In backoffice/views/courses_views.py
> <https://github.com/openfun/fun-apps/pull/520#discussion_r65030720>:
>
> > @@ -144,6 +144,22 @@ def course_detail(request, course_key_string):
> >
> >
> >  @group_required('fun_backoffice')
> > +def users(request, course_key_string, action=None):
> > +    course = get_course(course_key_string)
> > +    course_info = get_course_infos([course])[0]
> > +
> > +    users = User.objects.filter(courseenrollment__course_id=course.id).order_by('username')
>
> Je te confirme, il faut une pagination et même un fonction de recherche.
> Pour l'export c'est une bonne idée, faudra consulter pour savoir ce qu'il
> serait pertinent d'exporter
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openfun/fun-apps/pull/520/files/fc6253df21d44aaedab1425da6237b79f331341c#r65030720>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADTLO7c1x4qS2KVB8mTkZu6565Gxh82Bks5qGoljgaJpZM4IopRW>
> .
>